src/wanderer.ts

```diff
@@ -146,7 +159,7 @@ function freeCrafts() {
 }
 class WandererTarget {
   name: string;
-  available: () => boolean;
+  available: (type: DraggableFight) => boolean;
```

Comment:
This is making me start to wonder if going through the extant wanderer setup is worth it, given how different from everything else this is. Can we use the skipList and related functions without passing through this framework?

Actually this is helpful, comparing YR value to lovebug coinspiracy value is useful. Bleh.